internal/api/drafts.go

```diff
@@ -464,15 +451,15 @@ func DraftsHandler(
 						DocumentType:       docObj.GetDocType(),
 						DocumentTitle:      docObj.GetTitle(),
 						DocumentURL:        docURL,
-						DocumentProdAbbrev: docObj.GetProduct(),
-						DocumentTeamAbbrev: docObj.GetTeam(),
+						DocumentProd:       docObj.GetProduct(),
+						DocumentTeam:       docObj.GetTeam(),
 					}, emails,
 					)
 					//handle error gracefully
 					if err != nil {
 						fmt.Printf("Some error occured while sendind the message: %s", err)
 					} else {
-						fmt.Println("Succesfully! Delivered the message to all contributors")
+						fmt.Println("Succesfully! Delivered the EMAIL AND SLACK messageS to all contributors")
 					}
```

Comment:
why message contains capital letters in the middle of word, although it is not a issue.